authorization-common-api/src/main/java/com/sequenceiq/authorization/resource/AuthorizationResourceAction.java

```diff
@@ -21,6 +21,7 @@
     ACCESS_ENVIRONMENT("environments/accessEnvironment", AuthorizationResourceType.ENVIRONMENT),
     ADMIN_FREEIPA("environments/adminFreeIPA", AuthorizationResourceType.ENVIRONMENT),
     REPAIR_FREEIPA("environments/repairFreeIPA", AuthorizationResourceType.ENVIRONMENT),
+    SCALE_FREEIPA("environments/scaleFreeIPA", AuthorizationResourceType.ENVIRONMENT),
```

Comment:
please link the related commit/PR from thunderhead about the addition of this new permission

Reply:
https://github.infra.cloudera.com/thunderhead/thunderhead/pull/7020